rudder-core/src/main/scala/com/normation/rudder/repository/ldap/LDAPEntityMapper.scala

```diff
@@ -558,6 +583,9 @@ class LDAPEntityMapper(
     entry +=! (A_PRIORITY, directive.priority.toString)
     entry +=! (A_IS_ENABLED, directive.isEnabled.toLDAPString)
     entry +=! (A_IS_SYSTEM, directive.isSystem.toLDAPString)
+    for {
+      mode <- directive.policyMode
+    }  entry +=! (A_POLICY_MODE, mode.name)
```

Comment:
It's foreach syntax with a for 

I don't need the result, so don't need yield/flatmap or map